test/ctia/bundle/routes_test.clj

```diff
@@ -1125,3 +1125,42 @@
          (is (= 201 (:status link-res)))
          (is (= 200 (:status bundle-get-res)))
          (is (seq (-> bundle-get-res :parsed-body :casebooks))))))))
+
+(comment
+  (deftest race-condition-test
```

Comment:
```suggestion
  (deftest ^:disabled race-condition-test
```

Usually we use the [`disabled`](https://github.com/threatgrid/ctia/blob/32857663cdd7ac385161103dbafa8dc4f98febf0/dev-resources/circleci_test/config.clj#L19) selector to disable a test